org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/contentassist/CompletionProposalRequestor.java

```diff
@@ -713,6 +713,8 @@ private boolean matchCase(CompletionProposal proposal) {
 		if (proposal.getKind() == CompletionProposal.TYPE_REF) {
 			String simpleTypeName = SignatureUtil.getSimpleTypeName(proposal);
 			firstCharOfCompletion = simpleTypeName.charAt(0);
+		} else if (proposal.getKind() == CompletionProposal.METHOD_DECLARATION) {
```

Comment:
Yup, that's the one. I think this is correct. I'll just double check like always :stuck_out_tongue_winking_eye: